index.html

```diff
@@ -35,6 +35,16 @@
         <h1>Helping new talent into the tech world</h1>
       </div>
     </section>
+        <!--Business Partners Section-->
+
+        <section class="partners">
+          <div class="partners h2">
```

Comment:
You're using the `partners` class twice, on nested tags. This is confusing, and could lead to weird bugs later. For example, imagine you decide to style that class with a border:

```css
.partners {
  border: solid 1px #ccc;
}
```

It will apply to both the `<section>` and the `<div>`, and they will each have their own separate border. Probably not the intended behaviour.

Also, with your current CSS code, `.partners h2 { … }` will only be applied to the `<h2>` tag. So the `<div class="partners h2">` doesn't do anything useful. As a rule of thumb, if some code doesn't serve any purpose, just remove it.